packages/augur-ui/src/modules/market/components/market-view/betting-market-view.tsx

```diff
@@ -87,6 +87,11 @@ const BettingMarketView = () => {
     return () => (isMounted = false);
   }, [showCopied]);
 
+  useEffect(() => {
+    document.getElementById("mainContent")?.scrollTo(0, 0);
```

Comment:
so for this one, I think we need to find a solution because we are removing functionality. the idea here is we make sure to scroll back to the top of the page when we first hit market page cuz sometimes you can end up clicking into a market and being scrolled down the page etc.